controller/jobs.go

```diff
@@ -243,6 +243,21 @@ func (c *controllerAPI) RunJob(ctx context.Context, w http.ResponseWriter, req *
 		return
 	}
 
+	artifacts := make([]*ct.Artifact, len(artifactIDs))
+	artifactList, err := c.artifactRepo.ListIDs(artifactIDs...)
+	if err != nil {
+		respondWithError(w, err)
+		return
+	}
+	for i, id := range artifactIDs {
```

Comment:
why do need to make a copy of `artifactList` into `artifacts`. why not just use `artifactList`?